extensions/classification.md

```diff
@@ -0,0 +1,35 @@
+# Information Classification
+
+As part of ISO27001 control objective A8.2 aims to address 'Information Classification' whereby information and data in an organisation is properly managed, including classifcation in relation to sensitivity of the data, legislation etc. A.8.2.2 requires that electronic assets should be 'labelled', and this extension allows the `data` of cloudevents to be appropriately labelled with the `classification` of the event being shared.
```

Comment:
please wrap at 80 columns

Reply:
s/classifcation/classification/, and if we're doing en-US, it's "organization" and "labeled".